specification/compute/resource-manager/Microsoft.Compute/stable/2018-10-01/compute.json

```diff
@@ -6071,6 +6074,23 @@
       },
       "description": "Specifies the operating system settings for the virtual machine."
     },
+    "AutomaticRepairsPolicy": {
```

Comment:
>"AutomaticRepairsPolicy": { [](start = 4, length = 27)

this is modifying the existing api version.
please ensure that the default : false is honored for existing resources  (that don't have this property).
signing off, hoping this will be taken care.